### PR TITLE
DDT-608: Reference rename

### PR DESCRIPTION
**Pull request by mp-assco**

### CODE REVIEW

**Review by HenrikHL — DISMISSED** [review]
The name doesn't have to be escaped any more I guess - but no harm done...

File: src/main/java/org/dcsa/core/events/repository/ServiceRepository.java [hunk 1]
```diff
@@ -39,7 +39,7 @@
           + " ON st.transport_id = t.id "
           + "JOIN cargo_item ci "
           + " ON ci.shipment_id = st.shipment_id "
-          + "LEFT JOIN \"references\" r"
+          + "LEFT JOIN \"reference\" r"
           + " ON r.shipment_id = st.shipment_id"
           + "WHERE (ci.shipping_instruction_id = :shippingInstructionID OR references.shipping_instruction_id = :shippingInstructionID)")
```
Comment (HenrikHL) [inline src/main/java/org/dcsa/core/events/repository/ServiceRepository.java:44]:
```suggestion
          + "WHERE (ci.shipping_instruction_id = :shippingInstructionID OR reference.shipping_instruction_id = :shippingInstructionID)")
```

File: src/main/java/org/dcsa/core/events/repository/VoyageRepository.java [hunk 2]
```diff
@@ -32,7 +32,7 @@
           " ON st.transport_id = t.id " +
           "JOIN cargo_item ci " +
           " ON ci.shipment_id = st.shipment_id " +
-          "LEFT JOIN \"references\" r" +
+          "LEFT JOIN \"reference\" r" +
           " ON r.shipment_id = st.shipment_id" +
           "WHERE (ci.shipping_instruction_id = :shippingInstructionID OR references.shipping_instruction_id = :shippingInstructionID)")
```
Comment (HenrikHL) [inline src/main/java/org/dcsa/core/events/repository/VoyageRepository.java:37]:
```suggestion
          "WHERE (ci.shipping_instruction_id = :shippingInstructionID OR reference.shipping_instruction_id = :shippingInstructionID)")
```

**Comment by HenrikHL** [comment]
I found 2 places in this PR that needs renaming. Please make a global search - there might be other places as well...

**Comment by nt-as** [comment]
> The name doesn't have to be escaped any more I guess - but no harm done...

I would say we should remove the quotes because it _can_ cause issues.  By default, the database case normalizes unquoted input and accordingly the "public wisdom" is to either _always_ quote or _never_ quote.  Except for `"refereces"` we have never quoted, so I think we should standardize on "not quoting" now that we have the option (this implies that we should not quote the name in the SQL either in case we did that).

**Comment by mp-assco** [comment]
> > The name doesn't have to be escaped any more I guess - but no harm done...
> 
> I would say we should remove the quotes because it _can_ cause issues. By default, the database case normalizes unquoted input and accordingly the "public wisdom" is to either _always_ quote or _never_ quote. Except for `"refereces"` we have never quoted, so I think we should standardize on "not quoting" now that we have the option (this implies that we should not quote the name in the SQL either in case we did that).

@nt-as I have removed the quotations.

**Review by HenrikHL — APPROVED** [review]

**Pull request merged**